### PR TITLE
Add new variable NONROOT

### PR DESCRIPTION
**Pull request by xpivarc**
It can be used in test.sh to focus nonroot tests

Signed-off-by: L. Pivarc <lpivarc@redhat.com>

### CODE REVIEW

**Comment by xpivarc** [comment]
/cc @fgimenez @rmohr 

Used here https://github.com/kubevirt/kubevirt/pull/6112/commits/70125c0e5385cfb135a678b788fe5843953f0ef3

**Review by fgimenez — COMMENTED** [review]
/lgtm

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -801,6 +809,8 @@ presubmits:
           value: 1.13.8
         - name: FEATURE_GATES
           value: NonRoot
+        - name: NONROOT
```
Comment (rmohr) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:812]: Do we now have a `NONROOT` variable which is picked up in `kubevirt/kubevirt`? If so, It should probably have been named KUBEVIRT_NONROOT.

Comment (jean-edouard) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:812]: https://github.com/kubevirt/kubevirt/pull/6112 will use it.
I agree `KUBEVIRT_NONROOT` is a better name.
/hold

Comment (jean-edouard) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml]: /hold cancel

**Pull request merged**